src/components/Graph.react.js

```diff
@@ -212,6 +219,12 @@ PlotlyGraph.propTypes = {
      */
     clickData: PropTypes.object,
 
+    /**
+     * If True, `clear_on_doubleclick` will clear the `clickData` property
+     * when the user double clicks (when dragmode='zoom').
```

Comment:
Only on `dragmode='zoom'`? Also, could we be more specific about `dragmode`, like `figure['layout']['dragmode'] = 'zoom'` or `config['dragmode'] = 'zoom'`